.travis.yml

```diff
@@ -38,6 +38,6 @@ matrix:
     #####################
     ### DOCKER IMAGES ###
     #####################
-    - language: bash
-      script:
-        - make docker-build-jupyterlab
\ No newline at end of file
+    # - language: bash
```

Comment:
Remove these lines

Reply:
Removed in ccc1c8d0e41b4c4efd4f58c14887b2bde0e51a27.